src/main/java/de/komoot/photon/query/RequestBase.java

```diff
@@ -8,6 +8,7 @@ public class RequestBase {
     private String language = "default";
     private int limit = 15;
     private boolean debug = false;
+    private boolean dedupe = true;
```

Comment:
The parameter makes only sense for forward search, so the code from this file should go into [SearchRequestBase.java](https://github.com/komoot/photon/blob/master/src/main/java/de/komoot/photon/query/SearchRequestBase.java)

Reply:
@lonvia, ok, I'll move it. The reason why I added this parameter globally because StreetDupesRemover used along all requests